CRM/Core/xml/Menu/Tag.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="iso-8859-1" ?>
+
+<menu>
+  <item>
+    <path>civicrm/tag</path>
+    <title>Tags (Categories)</title>
+    <desc>Tags are useful for segmenting the contacts in your database into categories (e.g. Staff Member, Donor, Volunteer, etc.). Create and edit available tags here.</desc>
+    <page_callback>CRM_Tag_Page_Tag</page_callback>
+    <access_arguments>administer CiviCRM;manage tags</access_arguments>
+    <adminGroup>Customize Data and Screens</adminGroup>
```

Comment:
in PR #9126 eileenmcnaughton suggested <adminGroup> line should probably go

Answer: 
They are two navigation entries to manage tags, one in "Contacts" and another one in "Admin>Customize Data and Screens"
I have left it for the second one